apps/storybook/src/SvgElement.stories.tsx

```diff
@@ -0,0 +1,69 @@
+import {
+  DataToHtml,
+  Pan,
+  ResetZoomButton,
+  SvgElement,
+  SvgLine,
+  SvgRect,
+  VisCanvas,
+  Zoom,
+} from '@h5web/lib';
+import type { Meta, Story } from '@storybook/react';
+import { Vector3 } from 'three';
+
+import styles from './SvgElement.stories.module.css';
+import FillHeight from './decorators/FillHeight';
+
+export const Default: Story = () => {
+  return (
+    <VisCanvas
+      abscissaConfig={{ visDomain: [0, 10], showGrid: true }}
+      ordinateConfig={{ visDomain: [0, 10], showGrid: true }}
+    >
+      <Pan />
+      <Zoom />
+      <ResetZoomButton />
+
+      <DataToHtml
+        points={[
+          new Vector3(2, 8),
+          new Vector3(4, 6),
+          new Vector3(3, 2),
+          new Vector3(6, 4),
+          new Vector3(6, 6),
+          new Vector3(7, 7),
+        ]}
+      >
+        {(pt1, pt2, pt3, pt4, pt5, pt6) => (
+          <SvgElement>
```

Comment:
Nope, `DataToHtml` cannot be inside `SvgElement` since it requires access to the R3F context. `SvgElement` changes the renderer so we lose access to that context. You can have multiple `SvgElement` and wrap each of them with a `DataToHtml`, though. It's the same problem as with `Html`.